extensions/vscode/src/api/types/error.ts

```diff
@@ -1,12 +1,17 @@
 // Copyright (C) 2023 by Posit Software, PBC.
 
+import { ErrorCode } from "../../utils/errorTypes";
+
 type AgentErrorBase = {
-  code: string;
+  code: ErrorCode;
```

Comment:
Can we move this type under the API module since it is errors that are coming from our API? Feel free to ignore, just a cleanup suggestion.

Reply:
Agreed 100% with you, there are some more changes to do on error types and at some point we will consolidate these, hopefully in the near term